pkgs/development/libraries/arrow-cpp/default.nix

```
@@ -171,11 +171,39 @@ stdenv.mkDerivation rec {
         "TestFilterKernelWithNumeric/7.CompareArrayAndFilterRandomNumeric"
         "TestCompareKernel.PrimitiveRandomTests"
       ] ++ lib.optionals enableS3 [
+        "S3OptionsTest.FromUri"
+        "S3RegionResolutionTest.NonExistentBucket"
         "S3RegionResolutionTest.PublicBucket"
         "S3RegionResolutionTest.RestrictedBucket"
-        "S3RegionResolutionTest.NonExistentBucket"
-        "S3OptionsTest.FromUri"
         "TestMinioServer.Connect"
+        "TestS3FS.OpenOutputStreamBackgroundWrites"
+        "TestS3FS.OpenOutputStreamDestructorBackgroundWrites"
+        "TestS3FS.OpenOutputStreamDestructorSyncWrite"
+        "TestS3FS.OpenOutputStreamDestructorSyncWrites"
+        "TestS3FS.OpenOutputStreamMetadata"
+        "TestS3FS.OpenOutputStreamSyncWrites"
+        "TestS3FSGeneric.CopyFile"
+        "TestS3FSGeneric.CreateDir"
+        "TestS3FSGeneric.DeleteDir"
+        "TestS3FSGeneric.DeleteDirContents"
+        "TestS3FSGeneric.DeleteFile"
+        "TestS3FSGeneric.DeleteFiles"
+        "TestS3FSGeneric.DeleteRootDirContents"
+        "TestS3FSGeneric.GetFileInfo"
+        "TestS3FSGeneric.GetFileInfoAsync"
+        "TestS3FSGeneric.GetFileInfoGenerator"
+        "TestS3FSGeneric.GetFileInfoSelector"
+        "TestS3FSGeneric.GetFileInfoSelectorWithRecursion"
+        "TestS3FSGeneric.GetFileInfoVector"
+        "TestS3FSGeneric.MoveFile"
+        "TestS3FSGeneric.OpenInputFile"
+        "TestS3FSGeneric.OpenInputFileAsync"
+        "TestS3FSGeneric.OpenInputFileWithFileInfo"
+        "TestS3FSGeneric.OpenInputStream"
+        "TestS3FSGeneric.OpenInputStreamAsync"
+        "TestS3FSGeneric.OpenInputStreamWithFileInfo"
+        "TestS3FSGeneric.OpenOutputStream"
+        "TestS3FSGeneric.SpecialChars"
```

I believe, gtest supports globbing.
```suggestion
        "TestS3FSGeneric.*"
```